### PR TITLE
Pricing: highlight advanced plan

### LINKED ISSUE #288: Pricing: highlight "Advanced plan"
We want to add some kind of distinction for the "Advanced plan". Show it with a different color or similar. Besides, add a "Recommended" or "Most valued" or similar...

This is the page we are talking about https://about.readthedocs.com/pricing/

### PR DESCRIPTION
**Pull request by humitos**
Initial proposal to highlight advanced plan.

Screenshot_2024-06-16_23-17-38

Closes #288

----
📚 Documentation preview 📚: https://readthedocs-about--295.org.readthedocs.build/

### CODE REVIEW

**Comment by humitos** [comment]
I still think the ribbon version is a lot better, but I wasn't able to deal with the vertical spacing/alignment as @agjohnson anticipated 😅 

Screenshot_2024-06-17_09-45-51

Is there a simple way to solve this?

**Comment by agjohnson** [comment]
Yup, that is the issue I mentioned you would run into trying to use the ribbon style. There isn't a way around this, as it takes up vertical space.

**Comment by ericholscher** [comment]
Could put the ribbon above the pricing box, and just move them all down. But yea, not 100% sure the best option.

**Comment by agjohnson** [comment]
The ribbon style only really works if all the cells in the row have a similar ribbon, which makes this note not stand out. I had assigned this issue to myself to play with some better options here, but think we can start with something basic instead.

File: content/partials/pricing.html [hunk 1]
```diff
@@ -33,6 +33,12 @@ <h2 class="ui small monospace center aligned header">
 
       {{ caller() }}
 
+        {% if highlight %}
+        <div class="ui teal bottom attached label">
+          Most popular
```
Comment (agjohnson) [inline content/partials/pricing.html:38]: I'd make this at least more wordy.

```suggestion
          Our most popular plan
```

You might also add an icon to improve the visibility of this though.

**Comment by ericholscher** [comment]
Yea, I think the original concept is a reasonable first step, or a border or similar. Lots of options, just need to ship a v1 and then iterate when we have other ideas 👍

**Comment by humitos** [comment]
Screenshot_2024-06-18_10-35-21

**Review by agjohnson — APPROVED** [review]

**Pull request merged**